doc/_static/petab_schema.yaml

```diff
@@ -38,13 +38,24 @@ properties:
         files and optional visualization files.
       properties:
 
-        sbml_files:
-          type: array
-          description: List of PEtab SBML files.
-
-          items:
-            type: string
-            description: PEtab SBML file name or URL.
+        models:
+          type: object
+          description: One or multiple models
+
+          # the model ID
+          patternProperties: ^[a-zA-Z_]\w*$
```

Comment:
Could allow hyphens, but maybe problematic as the first character in YAML

Reply:
Could do that, or even allow for arbitrary strings. The current pattern is the same as for any other PEtab-related IDs, which would be easiest to remember.

Reply:
Ah, agreed :+1: